rules/windows/privilege_escalation_unusual_svchost_childproc_know_excl.toml

```diff
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2020/10/13"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies unusual child processes of svchost services that normally spawn limited child processes. This may indicate a
```

Comment:
Instead of using the word "limited" which may require interpretation, could we use "temporary" or "short-lived"?

Reply:
agree, this rule is not yet ready for shipping, prefer to delay it a bit (that's why 7.11 label applied)